.github/ISSUE_TEMPLATE/1_ask-a-question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Ask a question
```

Comment:
Please capitalize the header row of the table: about, title, etc.

Please add a period to the end of full sentences: Use this template...

Reply:
Done.